Competition2017/src/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -145,6 +145,20 @@ public void tankDriveVelocityPid(double leftPower, double rightPower) {
         updatePeriodicData();
     }
     
+    public void tankDriveVelocityInchesPerSec(double leftInchesPerSec, double rightInchesPerSec) {
+        // TODO: Move parameter updates to something more consistent
+        ensureSpeedModeForDrive();
+        
+        updateMotorConfig(leftDrive);
+        updateMotorConfig(rightDrive);
+
+        //Converting inches per second to ticks per 10ms
+        leftDrive.set(convertInchesToTicks(leftInchesPerSec)/100.0);
```

Comment:
you say 10ms in the comment above, but the code here is dividing by 100ms. Which is right?